crates/astria-celestia-jsonrpc-client/Cargo.toml

```diff
@@ -9,10 +9,14 @@ edition = "2021"
 base64.workspace = true
 base64-serde = "0.7.0"
 http = "0.2.9"
-jsonrpsee = { version = "0.18.2", features = ["macros", "client"] }
+
+jsonrpsee = { workspace = true, features = ["macros", "client"] }
 serde = { workspace = true, features = ["derive"] }
 serde_json = { workspace = true, features = ["raw_value"] }
 
+[features]
+server = ["jsonrpsee/server"]
```

Comment:
what's the server feature for? i guess for testing?

Reply:
Yeah, see the `dev-dependencies` entry for sequencer-relayer which uses it to mock a server